### DISCUSSION TITLE
Let eveyone submit their first finding to codehawks First Flight #2: Puppy Raffle

### DISCUSSION THREAD

**Discussion by rube-de** [Cyfrin/security-and-auditing-full-course-s23#105] [Ideas]
Since the First Flight #2: Puppy Raffle is closed, the course is kind of outdated, and the section about submitting a finding codehawks is unsatisfying as student.
This is especially observable when there is no running first flight or the first flight is expecting you to use new tools (just short time after telling about to make a break because of overloading with new info).

Why aren't you just let the First Flight #2: Puppy Raffle open indefinitely that everyone can have the same experience going through the course. 
(You could make a warning that this doesn't give XP anymore or give some symbolic XP automatically to encourage the students of the course)

**Comment by PatrickAlphaC** [comment]
Great recommendation:) 

We are working on that now!
